build.sh

```diff
@@ -184,6 +186,10 @@ while [[ $# -gt 0 ]]; do
         ICU_PATH="-DNO_ICU_PATH_GIVEN_SH=1"
         ;;
 
+    --no-jit)
+        NO_JIT="-DNO_JIT=1"
```

Comment:
@obastemur Agreed, changed 3 variables (all added by me) to `_sh` suffix in latest update.